### PR TITLE
Remove *View.Unregister, add stats.RegisterView for API consistency

### LINKED ISSUE #250: stats: add UnregisterView and then remove *View.Unregister for parity of APIs
I was just reviewing https://github.com/census-instrumentation/opencensus-go/pull/249/files and noticed that the stats API has
stats.RegisterView
but no matching call to stats.UnregisterView

and that instead the caller has to do *View.Unregister(), of which there also isn't a match method *View.Register()

Notice that RegisterView and *View.UnregisterView look like the perfect match
https://github.com/census-instrumentation/opencensus-go/blob/86bea64327bd7d353334ff858a2e6675a3122b7b/stats/worker.go#L115-L122
and
https://github.com/census-instrumentation/opencensus-go/blob/86bea64327bd7d353334ff858a2e6675a3122b7b/stats/worker.go#L127-L134

I'd say let's deprecate/remove *View.Unregister and instead add the UnregisterView helper since
we already have *View.Subscribe and *View.Unsubscribe of which Subscribe and Unsubscribe logically use RegisterView and UnregisterView respectively.

### PR DESCRIPTION
**Pull request by odeke-em**
Fixes #250

### CODE REVIEW

File: internal/readme/stats.go [hunk 1]
```diff
@@ -99,7 +99,7 @@ func statsExamples() {
 	_ = v
 
 	// START unregisterView
-	if v.Unregister(); err != nil {
+	if stats.UnregisterView(v); err != nil {
```
Comment (songy23) [inline internal/readme/stats.go]: Can you also run embedmd to re-generate the root README file?

Comment (odeke-em) [reply internal/readme/stats.go]: Ah yes, thanks for that catch. Let me do so.

Comment (odeke-em) [reply internal/readme/stats.go]: @songy23 done, PTAL.

**Comment by odeke-em** [comment]
Thanks y'all for the review!

**Pull request merged**